libs/design-system/logos/apps/src/lib/app-logos.component.scss

```diff
@@ -0,0 +1,116 @@
+:host {
+  display: flex;
+  height: 2.375rem;
+
+  &[variant='fixed'] {
+    height: 3rem;
+
+    .hra-logo {
+      margin-right: 0.25rem;
+      width: unset;
+
+      .hra-logo-text {
+        display: none;
+      }
+    }
+
+    .app-logo {
+      img {
+        margin-right: 0.75rem;
+      }
+
+      .app-label {
+        font-size: 1.25rem;
+        line-height: 1.875rem;
+      }
+
+      .app-description {
+        font-size: 0.875rem;
+      }
+    }
+  }
+
+  &[no-description='true'] {
+    .app-logo .app-description {
+      display: none;
```

Comment:
The `@if` statement will remove the element from the dom so this has no effect. Please remove